runtime/near-vm-runner/src/wasmer1_runner.rs

```diff
@@ -87,26 +90,140 @@ impl IntoVMError for wasmer::InstantiationError {
     }
 }
 
+// Copy-pasted, shall be exported by Wasmer instead.
```

Comment:
Hm, it is public? https://docs.rs/wasmer-vm/1.0.2/wasmer_vm/enum.TrapCode.html

Reply:
Indeed, fixed.

Reply:
Indeed, fixed.